src/ensembl/src/content/app/browser/drawer/drawer-views/gene-summary/GeneSummary.tsx

```diff
@@ -16,30 +16,138 @@
 
 import React from 'react';
 import { useSelector } from 'react-redux';
+import { gql, useQuery } from '@apollo/client';
 
-import { getDisplayStableId } from 'src/shared/state/ens-object/ensObjectHelpers';
+import * as urlFor from 'src/shared/helpers/urlHelper';
+import { getFormattedLocation } from 'src/shared/helpers/formatters/regionFormatter';
+import { getStrandDisplayName } from 'src/shared/helpers/formatters/strandFormatter';
+import {
+  buildFocusIdForUrl,
+  getDisplayStableId
+} from 'src/shared/state/ens-object/ensObjectHelpers';
 import { getBrowserActiveEnsObject } from 'src/content/app/browser/browserSelectors';
 
+import ViewInApp from 'src/shared/components/view-in-app/ViewInApp';
+
 import { EnsObjectGene } from 'src/shared/state/ens-object/ensObjectTypes';
+import { Gene as GeneFromGraphql } from 'src/shared/types/thoas/gene';
+
+import styles from './GeneSummary.scss';
+
+const GENE_QUERY = gql`
+  query Gene($genomeId: String!, $geneId: String!) {
+    gene(byId: { genome_id: $genomeId, stable_id: $geneId }) {
+      alternative_symbols
+      name
+      stable_id
+      symbol
+      so_term
+      transcripts {
+        stable_id
+      }
+      slice {
+        strand {
+          code
+          value
+        }
+        location {
+          length
+        }
+      }
+    }
+  }
+`;
 
-import styles from 'src/content/app/browser/drawer/Drawer.scss';
+type Gene = Required<
+  Pick<
+    GeneFromGraphql,
+    | 'stable_id'
+    | 'symbol'
+    | 'name'
+    | 'alternative_symbols'
+    | 'so_term'
+    | 'transcripts'
+    | 'slice'
+  >
+>;
```

Comment:
As a side note: this type isn't really reflecting what you are getting from Thoas (e.g. you are definitely not getting the whole `transcripts`). Hopefully, this will be mitigated with the approach that's being developed in https://github.com/Ensembl/ensembl-client/pull/452.